### ISSUE TITLE
Can't start if daemon is on with systemd

### ISSUE THREAD

**Issue by deimosfr** [jdauphant/ansible-role-nginx#38]
Hi,

I'm trying to get it working with Debian Jessie. However, with systemd there's an issue when daemon is set to "on":
```
Feb 26 17:58:45 vm sudo[5522]: pam_unix(sudo:session): session closed for user root
Feb 26 17:58:45 vm sudo[5529]: vagrant : TTY=unknown ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo SUDO-SUCCESS-lkyorsvjl
Feb 26 17:58:45 vm sudo[5529]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 26 17:58:45 vm ansible-<stdin>[5531]: Invoked with name=nginx pattern=None enabled=True state=started sleep=None arguments= runle
Feb 26 17:58:45 vm nginx[5563]: nginx: [emerg] "daemon" directive is duplicate in /etc/nginx/nginx.conf:51
Feb 26 17:58:45 vm nginx[5563]: nginx: configuration file /etc/nginx/nginx.conf test failed
Feb 26 17:58:45 vm systemd[1]: nginx.service: control process exited, code=exited status=1
Feb 26 17:58:45 vm systemd[1]: Failed to start A high performance web server and a reverse proxy server.
-- Subject: Unit nginx.service has failed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit nginx.service has failed.
-- 
-- The result is failed.
Feb 26 17:58:45 vm systemd[1]: Unit nginx.service entered failed state.
Feb 26 17:58:45 vm sudo[5529]: pam_unix(sudo:session): session closed for user root
```

Trying to find the way to make it work

**Comment by deimosfr** [comment]
I've found the issue. On Debian Jessie, it looks like you simply can't disable the daemon. You can see the systemd configuration file (/lib/systemd/system/nginx.service) the override:
```
ExecStartPre=/usr/sbin/nginx -t -q -g 'daemon on; master_process on;'
ExecStart=/usr/sbin/nginx -g 'daemon on; master_process on;'
ExecReload=/usr/sbin/nginx -g 'daemon on; master_process on;' -s reload
ExecStop=-/sbin/start-stop-daemon --quiet --stop --retry QUIT/5 --pidfile /run/nginx.pid
```
So when the templates tries to add the daemon directive line, it fails.

I'm going to provide a pull request for it.

**Comment by deimosfr** [comment]
A pull request have been open to correct the issue #39

**Comment by jdauphant** [comment]
Ticket resolved by #39
